FXMLDocumentController.java

```diff
@@ -3,133 +3,423 @@
+            Statement stm= cnx.createStatement();
+            ResultSet res = stm.executeQuery(query);
+           
+            while(res.next())
+            {
+             cbxTicketAjouterRes.getItems().add(res.getInt("idTicket")+""); 
+            }
+           
+            System.out.println("combobox1 rempli!");
+            
+            Statement stm2= cnx.createStatement();
+            String query2 ="select * from alimentations";
+            ResultSet res2 = stm2.executeQuery(query2);
+            
+            while(res2.next())
+            {
+                cbxAlimAjouterRes.getItems().add(res2.getInt("idAlim")+""); 
+            }
+           
+            System.out.println("combobox2 rempli!");
+            
+        } catch (SQLException e) {
+            System.out.println("erreur combobox!");
+            System.out.println(e.getMessage());
+        }
+    }
+    
+    
+    private void remplirCbxModifierRes()
+    {    
+
+        cnx=Maconnexion.getInstance().getConnection();
+        try {
+            
+            String query ="select * from tickets";
+            Statement stm= cnx.createStatement();
+            ResultSet res = stm.executeQuery(query);
+           
+            while(res.next())
+            {
+             cbxTicketModifierRes.getItems().add(res.getInt("idTicket")+""); 
+            }
+           
+            System.out.println("combobox1 rempli!");
+            
+            Statement stm2= cnx.createStatement();
+            String query2 ="select * from alimentations";
+            ResultSet res2 = stm2.executeQuery(query2);
+            
+            while(res2.next())
+            {
+                cbxAlimModifierRes.getItems().add(res2.getInt("idAlim")+""); 
+            }
+           
+            System.out.println("combobox2 rempli!");
+            
+            
+        } catch (SQLException e) {
+            System.out.println("erreur combobox!");
+            System.out.println(e.getMessage());
+        }
     }
     
     @Override
     public void initialize(URL url, ResourceBundle rb) {
         // TODO
     }    
 
-   @FXML
-    private void ajouterpromotion(ActionEvent event) {
-         ServicePromotion ff=new ServicePromotion();
-          Promotion f=new Promotion();
-         f.setId(Integer.parseInt(tfid.getText()));
-         f.setNom(tfnom.getText());
-         f.setType(tftype.getText());
-         f.setDescription(tfdescription.getText());
-         f.setPourcentage(tfpourcentage.getText());
-         
-         
-        try {
-            ff.AddPromotion(f);
-        } catch (SQLException ex) {
+    @FXML
+    private void login(ActionEvent event) {
+        if(cbxTypeLogin.getValue().equals("Simple User"))
+        {
+            
+        }
+        else if(cbxTypeLogin.getValue().equals("Administrateur"))
+        {
+            
         }
-        
         
     }
-    
-    
 
+    @FXML
+    private void SignUp(ActionEvent event) {

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.